purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1828,7 +1828,7 @@ class Purchases internal constructor(
                     customerInfoHelper.sendUpdatedCustomerInfoToDelegateIfChanged(info)
                     onSuccess()
                 },
-                onError = { error, shouldConsumePurchase, body ->
+                onError = { error, shouldConsumePurchase, _, body ->
```

Comment:
Currently unused but we will need to use the new parameter to detect whether we need to compute offline entitlements or not.